app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    com_ls = command.split()
```

Comment:
Consider using unpacking for the command list to improve readability and adhere to the checklist's recommendation. For example: `_, sour_file, path = command.split()`.